modules/auxiliary/odt/badodt.rb

```diff
@@ -0,0 +1,99 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'base64'
+require 'rex/zip'
+require 'msf/core/auxiliary/report'
```

Comment:
I don't think this line is necessary, unless you're using [the functionality within the `report` module](https://www.rubydoc.info/github/rapid7/metasploit-framework/Msf/Auxiliary/Report).